CodeLanguages-Container/CodeLanguages-Container.xcodeproj/project.pbxproj

```diff
@@ -6,74 +6,11 @@
 	objectVersion = 56;
 	objects = {
 
-/* Begin PBXBuildFile section */
```

Comment:
There's a lot of deletions in the `project.pbxproj` file. Please make sure, that you correctly resolved all dependencies. There shouldn't be any deletions at all - just additions related to the `tree-sitter-toml` repo. Take a look at this PR for example: https://github.com/CodeEditApp/CodeEditLanguages/pull/28/files#diff-0a20baec64c8334a037d370407084a60dda04a6bee936c7eb1608656b26748cbR8-R684

Reply:
That's weird I don't remember deleting anything, I'll look into it

Reply:
You somehow deleted the framework target.

<img width="213" alt="Screenshot 2023-02-25 at 18 00 57" src="https://user-images.githubusercontent.com/9460130/221369831-697edcbf-f30e-4fb4-85c4-61dfa591a7ec.png">

And that's what it should look like:

<img width="196" alt="Screenshot 2023-02-25 at 18 02 09" src="https://user-images.githubusercontent.com/9460130/221369864-40f99720-20f7-4a12-8a61-019d15f2dc16.png">

Reply:
How did I do that? I'll try to restore that

Reply:
Therefore when running the `build_framework.sh` script, it won't succeed. When running it with the `--debug` flag you get the error `xcodebuild: error: The project named "CodeLanguages-Container" does not contain a scheme named "CodeLanguages-Container". The "-list" option can be used to find the names of the schemes in the project.`